### PR TITLE
Added config file path flag and Dockerfile

### PR DESCRIPTION
**Pull request by cristure**
Checked out from `hyperblock-processor`. Merge after that branch is also merged.

### CODE REVIEW

File: cmd/connector/main.go [hunk 1]
```diff
@@ -54,7 +57,17 @@ func main() {
 }
 
 func startConnector(ctx *cli.Context) error {
-	cfg, err := loadConfig(configPath)
+	configFilePath := ctx.GlobalString(configFile.Name)
+	if configFilePath == "" {
+		configFilePath = defaultConfigPath
+	}
```
Comment (ssd04) [inline cmd/connector/main.go:63]: default config path can be set as `Value` for `cli.StringFlag` and there is no need for this additional check

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Multiversx node
+WORKDIR /go/mx-chain-ws-connector-firehose-go/cmd/connector
+RUN go build -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty)"
+RUN mv connector /usr/bin/connector
+
+# ===== SECOND STAGE ======
+FROM ubuntu:22.04
+RUN apt-get update && apt-get upgrade -y
+COPY --from=builder /usr/bin/connector /app/connector
+WORKDIR /app
```
Comment (ssd04) [inline Dockerfile:15]: expose here default connector port `22111`, at least as a way of documenting?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1.20.7 as builder
+
+WORKDIR /go/mx-chain-ws-connector-firehose-go
+COPY . .
+RUN go mod tidy
+# Multiversx node
```
Comment (ssd04) [inline Dockerfile:6]: udpate or remove this comment since it is not related to mx-chain-go node

File: cmd/connector/main.go (hunk 1 above)
Comment (cristure) [reply cmd/connector/main.go:63]: done.

File: Dockerfile (hunk 2 above)
Comment (cristure) [reply Dockerfile:15]: done.

File: Dockerfile (hunk 3 above)
Comment (cristure) [reply Dockerfile:6]: done.

**Review by AdoAdoAdo — APPROVED** [review]

File: cmd/connector/main.go [hunk 4]
```diff
@@ -53,7 +57,14 @@ func main() {
 }
 
 func startConnector(ctx *cli.Context) error {
-	cfg, err := loadConfig(configPath)
+	configFilePath := ctx.GlobalString(configFile.Name)
+
+	absoluteConfigPath, err := filepath.Abs(configFilePath)
```
Comment (ssd04) [inline cmd/connector/main.go:62]: i don't think we need absolute path here, it should work also with relative path

Comment (cristure) [reply cmd/connector/main.go]: changed.

**Review by ssd04 — APPROVED** [review]

**Pull request merged**